OWNERS

```diff
@@ -4,10 +4,10 @@ approvers:
 - cjwagner
 - fejta
 - ixdy
+- Katharine
```

Comment:
context: is currently oncall :-) :+1:

Reply:
But we will or will not remove after... ?

Reply:
Planning to leave it until she removes herself